Barotrauma/BarotraumaShared/SharedSource/Items/Components/ItemContainer.cs

```diff
@@ -492,6 +491,12 @@ public override void Move(Vector2 amount, bool ignoreContacts = false)
 
         public void OnItemRemoved(Item containedItem)
         {
+            foreach (ActiveContainedItem activeContainedItem in activeContainedItems)
+            {
+                if (activeContainedItem.Item != containedItem || !TryProcessActiveContainedItem(activeContainedItem)) continue;
```

Comment:
Single-line if clauses should also use curved brackets:

`if (activeContainedItem.Item != containedItem || !TryProcessActiveContainedItem(activeContainedItem)) { continue; }`